model/delivery/LtiTestTakerAuthorizationService.php

```diff
@@ -110,4 +134,15 @@ public function isSuitable(User $user, $deliveryId = null)
         ], $user->getRoles());
         return !empty($ltiRoles);
     }
+
+    private function isAuthorizedLtiExecutionWithWrongUrl(DeliveryExecutionInterface $deliveryExecution): bool
+    {
+        return $this->getRequest()->getUri()->getPath() === '/ltiDeliveryProvider/DeliveryRunner/runDeliveryExecution'
```

Comment:
Suggestion: Maybe we can move this into a private class constant (`URL_DELIVERY_RUNNER_DELIVERY_EXECUTION` or something like that)?

Reply:
Yes, I can, but what would be the difference if is only used here and it is private?

Reply:
Not much , I was thinking this method may be a bit more readable if we moved the URL to a constant with a name that reflects what the URL is (`ltiDeliveryProvider/DeliveryRunner` being from DeliveryRunner instead of the one from this extension), but we can keep it as it is too.

Reply:
Still, the method name explains it and the string is only used here, so I do not see the need on having a constant for this. 

I will need to backport this code, so I am trying to avoid refactoring as much as I can in this case.